project/Dependencies.scala

```diff
@@ -54,13 +54,19 @@ object Dependencies {
     "com.mattbertolini" % "liquibase-slf4j" % "2.0.0"
   )
 
+  val coreDependencies = List(
+    "com.typesafe.akka" %% "akka-actor" % akkaV
+  ) ++ testDependencies
+
+  val backendDependencies = List(
+    "org.broadinstitute" %% "wdl4s" % wdl4sV
```

Comment:
Cromwell-backend needs from wdl4s to live... and Cromwell-engine will need from Cromwell-backend to live... then in the way the build mechanism is defined you have two options in order to not repeat the dependency entry:
1. As I defined.
2. Define a new list of dependencies with just wdl4s and add that to backend and engine.

In my opinion this should not be a big deal... we can define this using any of the two options.